### PR TITLE
Conditional compilation for unix tragets.

### PR DESCRIPTION
**Pull request by leonardodanelutti**

### CODE REVIEW

**Comment by maxmindlin** [comment]
Hey I quickly added CI/CD tests to master branch. Could you update your branches and push them up to allow the pipelines to run?

**Comment by leonardodanelutti** [comment]
Sure!

File: .idea/workspace.xml
Comment (maxmindlin) [inline .idea/workspace.xml:1]: can you remove this and add it to the .gitignore?

Comment (leonardodanelutti) [reply .idea/workspace.xml:1]: Yes, sorry, I forgot to delete it.

Comment (maxmindlin) [reply .idea/workspace.xml:1]: no worries, thanks

**Comment by maxmindlin** [comment]
Looks good! Thanks for the contribution and working with me while I setup CI/CD.

**Review by maxmindlin — APPROVED** [review]

**Pull request merged**